components/PageHeaderCard.tsx

```diff
@@ -1,43 +1,69 @@
-import {AspectRatio, Box, Stack, Typography} from '@mui/joy';
+import { AspectRatio, Box, Stack, Typography } from '@mui/joy';
 import Image from 'next/image';
 
-
 interface PageHeaderCardProps {
   imagePath: string;
   pageTitle: string;
 }
-export default function PageHeaderCard({imagePath, pageTitle} : PageHeaderCardProps) {
+
+export default function PageHeaderCard({ imagePath, pageTitle }: PageHeaderCardProps) {
   return (
-      <AspectRatio ratio='700/150' objectFit='cover'
-                   sx={{width: '100%', height: '300px', position: 'relative', overflow:'hidden'}}>
-        <Image src={imagePath} alt='page-header-image' fill/>
-        <Box
-          className='bg-dark-blue'
+    <AspectRatio
+      ratio="700/150"
+      objectFit="cover"
+      sx={{
+        width: '100%',
+        height: {
```

Comment:
You can also use the keyword 'auto' so that the height changes automatically based on the width